client/my-sites/site-settings/disconnect-site/style.scss

```diff
@@ -21,3 +21,11 @@
 .disconnect-site__navigation-links {
 	text-align: center;
 }
+
+// 'Missing Feature' page
+.disconnect-site__missing-feature {
+	.button {
```

Comment:
Minor: Rather than a `.button` descendent of the parent, how about targeting the button directly with a `className="disconnect-site__missing-feature-submit"`?

Reply:
Hmm, if I do that, the seemingly more generic `.button` selector overrides `margin` to be `0`...

Reply:
😢  Let's just leave it then, probably best as is because we'll need to add specificity somehow.